ads/getting-started/ios.mdx

```diff
@@ -31,54 +28,54 @@ import TabItem from '@theme/TabItem';
         ```sh
         pod install
         ```
-        </TabItem>
-       <TabItem value="swiftpm" label="SwiftPM">
+    </TabItem>
+    <TabItem value="swiftpm" label="SwiftPM">
          1. Open your Xcode project and navigate to File > Add Package Dependencies...
 
-         ![Screenshot of Xcode file menu](../assets/img/theoads-ios-xcode-file-menu.png)
+         ![Screenshot of Xcode file menu](../assets/img/ios-xcode-file-menu.png)
 
          2. Add `theoplayer-sdk-apple` package by entering the following url:
```

Comment:
nitpick: @FabienLavocat I know this wasn't a change you made, but should we capitalize URL here? It looks like there is a bit of a mix in the docs, after doing a quick scan. So, 🤷

Reply:
I am not sure what you would want to capitilize here?